src/query/ast/src/parser/statement.rs

```diff
@@ -1014,12 +1014,12 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
         rule! {
             CREATE ~ STAGE ~ ( IF ~ ^NOT ~ ^EXISTS )?
             ~ ( #stage_name )
-            ~ ( URL ~ ^"=" ~ ^#uri_location)?
+            ~ ( URL ~ ^"=" ~ ^#uri_location )?
```

Comment:
is this only for formatting or actually affective for parsing?

Reply:
it's just formatting